applications/ParticleMechanicsApplication/custom_elements/updated_lagrangian.cpp

```diff
@@ -1255,7 +1255,14 @@ void UpdatedLagrangian::CalculateDampingMatrix( MatrixType& rDampingMatrix, cons
     const unsigned int dimension = GetGeometry().WorkingSpaceDimension();
 
     //resizing as needed the LHS
-    unsigned int matrix_size = number_of_nodes * dimension;
+    unsigned int matrix_size;
+    if (rCurrentProcessInfo["pressure_dofs"]) {
+        matrix_size = number_of_nodes * (dimension + 1);
+    }
+    else {
+        matrix_size = number_of_nodes * dimension;
+    }
+    
```

Comment:
Why is this change needed here?

Reply:
Even if the merging was not in the master, I reply as promised (sorry for being late). The change is due to the fact that in the master version of the UP element a degree of freedom is added (pressure) but the mass and damping matrix dimensions were not updated by consequence causing errors in the dynamic simulations.